spoti-friends/src/friendActivity/views/ListeningActivityDetails.swift

```diff
@@ -3,18 +3,24 @@ import SwiftUI
 /// The View that renders the details for a listening activity item.
 ///
 /// - Parameters:
-///   - displayName: The display name for the user whose listenting activity this is.
-///   - track: The current or most recent track to display for the user.
+///   - profile: The Spotify Profile for the user whose listening activity this is.
+///   - currentTrack: The current or most recent track to display for the user.
+///   - track: The actual `Track` object for the `currentTrack`.
```

Comment:
Change to correct name
```suggestion
///   - trackDetails: The actual `Track` object for the `currentTrack`.
```